src/common/components/vanilla-modules/forms/form-group-select.js

```diff
@@ -0,0 +1,61 @@
+import React, { PropTypes } from 'react';
+
+import styles from '../../../style/vanilla/css/forms.css';
+
+export default function FormGroupSelect(props) {
+  const { disabled, name, label, placeholder } = props;
+  const id = `ID_${name}`;
+  const status = props.touched ? ( props.valid ? 'p-form-validation is-success' : 'p-form-validation is-error') : null;
+
+  return (
+    <div className={ `${styles['p-form__group']} ${styles[status]}` }>
+      <label htmlFor={ id } className={ styles['p-form__label'] }>
+        { label }
+      </label>
+      <div className={ styles['p-form__control'] }>
+        <select
+          id={ id }
+          name={ props.sensitive ? null : name }
+          data-name={ name }
+          required={ props.required }
+          disabled={ disabled }
+          onChange={ props.onChange }
+          value={ props.selectedOption }
+        >
+          <option value="">{ placeholder }</option>
```

Comment:
For a select dropdown I would prefer there was an option like this. As the placeholder attribute doesn't work with `<select>` this is the better option.